site/gatsby-site/src/components/users/UsersField.js

```diff
@@ -31,11 +31,13 @@ export default function UsersField({ id, name, placeHolder = '' }) {
   useEffect(() => {
     if (data?.users) {
       setSelected((selected) =>
-        selected.map(({ id }) => {
-          const { userId, first_name, last_name } = data.users.find((user) => user.userId == id);
+        selected
+          .filter(({ id }) => data.users.some((user) => user.userId == id))
```

Comment:
This may not be necessary. On other than dev environments, it would never happen 🤔